static/js/src/PurchaseModal/components/BuyButton/BuyButton.tsx

```diff
@@ -87,6 +87,7 @@ const BuyButton = ({ setError, quantity, product, action }: Props) => {
         onSuccess: (data) => {
           //start polling
           setPendingPurchaseID(data);
+          localStorage.clear();
```

Comment:
Maybe we should just clear pro related values ? so we don't mess up with whoever else could be using the local storage on ubuntu.com